docs/tutorials/mobile/pose-detection.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Object detection and pose estimation with Yolov8
+description: Run the Yolov8 model on mobile (iOS and Android) with built-in pre and post processing to perform object detection and pose estimation
+image: /images/pose-thumbnail.png
+parent: Deploy on mobile
+grand_parent: Tutorials
+nav_order: 1
+---
+
+# Object detection and pose estimation on mobile with Yolov8
```

Comment:
nit: Would be nice to have a TOC with links to Object Detection and Pose Estimation here.